main/py-requests/APKBUILD

```diff
@@ -2,12 +2,13 @@
 # Maintainer: William Pitcock <nenolod@dereferenced.org>
 pkgname=py-requests
 _pkgname=requests
-pkgver=2.13.0
+pkgver=2.16.5
 pkgrel=0
 pkgdesc="A HTTP request library for Python"
 url="http://www.python-requests.org/"
 arch="noarch"
-license="ASL-2.0"
+license="ASL 2.0"
+options="!check"
```

Comment:
Here the license matches the current [convention](https://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package#license).

Comment added